packages/wdio-browserstack-service/src/service.js

```diff
@@ -9,6 +9,8 @@ export default class BrowserstackService {
     constructor (config) {
         this.config = config
         this.failures = 0
+        this.fullTitle = ''
+        this.failReason = ''
```

Comment:
there is no reason to define them with empty string

Reply:
if I delete failReason, we will get undefined in _getBody()
we have 2 options:
1. we can leave it with empty string
2. check in _getBody() if we need reason field and only if required (test failed) we will send it with the this.failReason